src/Emulator/Peripherals/Peripherals/Timers/ATCPIT100.cs

```diff
@@ -0,0 +1,963 @@
+                    else{
+                        this.Log(LogLevel.Error, "Cannot enable interrupt of timers 0 & 1 when channel {0} is in {1} mode", 
+                            channelNum, ChannelN_Control_ChMode[channelNum]);
+                    }
+                    break;
+            }
+            this.InfoLog("setting ch{0} timer{1}'s interrupt value to {2} with channel mode {3}", 
+                channelNum, timerNum, interruptValue, (ChannelMode)ChannelN_Control_ChMode[channelNum]);
+            UpdateInterrupts();
+        }
+
+        private bool InterruptEnableReturn(int channelNum, int timerNum){
+            bool returnVal = false;
+            switch (ChannelN_Control_ChMode[channelNum]){
+                case ChannelMode.Timer_32bit:
+                    ChannelN_InterruptM_En[channelNum, timerNum] = internalTimers[channelNum, timerNum].Compare0Interrupt;
+                    returnVal = ChannelN_InterruptM_En[channelNum, timerNum];
+                    break;
+                case ChannelMode.Timer_16bit:
+                    ChannelN_InterruptM_En[channelNum, timerNum] = internalTimers[channelNum, timerNum + 1].Compare0Interrupt;
+                    returnVal = ChannelN_InterruptM_En[channelNum, timerNum];
+                    break;
+                case ChannelMode.Timer_8bit:
+                    ChannelN_InterruptM_En[channelNum, timerNum] = internalTimers[channelNum, timerNum + 3].Compare0Interrupt;
+                    returnVal = ChannelN_InterruptM_En[channelNum, timerNum];
+                    break;
+                case ChannelMode.PWM:
+                    //TODO: Set PWM interrupt value
+                    returnVal = false;
+                    break;
+                case ChannelMode.PWM_Timer_16bit:
+                    //TODO: Set PWM interrupt value
+                    ChannelN_InterruptM_En[channelNum, timerNum] = internalTimers[channelNum, timerNum].Compare0Interrupt;
+                    returnVal = ChannelN_InterruptM_En[channelNum, timerNum];
+                    break;
+                case ChannelMode.PWM_Timer_8bit:
+                    //TODO: Set PWM interrupt value
+                    ChannelN_InterruptM_En[channelNum, timerNum] = internalTimers[channelNum, timerNum + 1].Compare0Interrupt;
+                    returnVal = ChannelN_InterruptM_En[channelNum, timerNum];
+                    break;
+                default:
+                    this.Log(LogLevel.Error, "Error reading ch{0} timer{1}'s interrupt enable. Illegal ChannelMode: {3}", channelNum, timerNum, ChannelN_Control_ChMode[channelNum]);
+                    returnVal = false;
+                    break;
+            }
+            //UpdateInterrupts();
+            return returnVal;
+        }
+
+        private void InterruptStatus(int channelNum, int timerNum, bool value){
+            //value is inverted by W1C control in register before function call
+            //Console.WriteLine("intstatus() value: {0}" , value);
```

Comment:
Also, delete any logging commented. Keep comments to provide insight/information regarding what was implemented.